src/test/apiTest/java/uk/gov/hmcts/ethos/replacement/apitest/utils/CCDRequestBuilder.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.ethos.replacement.apitest.utils;
+
+import uk.gov.hmcts.et.common.model.ccd.CCDRequest;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.CaseDetails;
+
+public class CCDRequestBuilder {
```

Comment:
Could we not reuse the class for the unit tests rather than duplicate?

Reply:
I will create new story for this and merge it as a separate PR as discussed.

Reply:
[RET-3120](https://tools.hmcts.net/jira/browse/RET-3120)